tensorflow/core/framework/tensor.cc

```diff
@@ -677,14 +677,43 @@ bool Tensor::CanUseDMA() const {
 #undef CASE
 
 namespace {
+// Recursive print from left dim to right dim
```

Comment:
add punctuation + "Print ... recursively." is probably better phrasing, or Print recursively, from....